CRM/Activity/BAO/Activity.php

```diff
@@ -2799,7 +2817,7 @@ protected static function hasPermissionForActivityType($activityTypeID) {
    *
    * @return array
    */
-  public static function getPermittedActivityTypes() {
+  protected static function getPermittedActivityTypes() {
```

Comment:
What's the reason for changing the visibility of this function? That has a potential breaking impact in case any extension relies on access to it.

Reply:
@colemanw it's recently added so it's to prevent anyone starting to use it since we are still in a time frame where we can assume they haven't. (making things protected is one way to discourage unsupported integration practices)

Reply:
Ok